app/main.py

```diff
@@ -1,8 +1,48 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```

Comment:
Place each argument on a new line including self, so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability. Apply this to `__init__` for `CarWashStation` class as well.